service.yml

```diff
@@ -0,0 +1,11 @@
+name: kafka-rest
+lang: unknown
```

Comment:
@cprovencher should language be Java with a specific version?

Reply:
Already answered here https://github.com/confluentinc/ce-kafka-rest/pull/548#discussion_r1387151749